layouts/guest/single.html

```diff
@@ -108,7 +108,17 @@ <h3>Episodes Featuring {{ .Title }}</h3>
             {{- range $page := (where ( where site.RegularPages "Type" "in" site.Params.mainSections) ".Params.upcoming" "!=" true ) -}}
               {{- range $page.Params.guests -}}
                 {{- if eq  . ($.Scratch.Get "guest-name") -}}
-                  <a href = "{{$page.Permalink}}" class = "guest_page_episode_link list-group-item list-group-item-action">{{$page.Title}}</a>
+                  {{ if isset $.Site.Params "episode_number_style"}}
```

Comment:
add the check for .Params.episode here as well (basically the same if AND statement)